x/gov/abci.go

```diff
@@ -185,8 +185,15 @@ func EndBlocker(ctx sdk.Context, k *keeper.Keeper) error {
 			return false, err
 		}
 
-		if !quorumReached {
+		// quorum not reached and the proposal is not expired; schedule the next tally
```

Comment:
!ctx.BlockTime().After(proposal.VotingEndTime)? it seems to finish proposal when block time reached the exact VotingEndTime